### PR TITLE
[DISCUSSION] docs: Add Code style suggestions, in particular for use statements

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: docs/Contributing.asciidoc [hunk 1]
```diff
@@ -76,6 +76,17 @@ implications and how we can help each other to improve
 If this is too much hassle for you feel free to provide incomplete pull
 requests for consideration or create an issue with a code change proposal.
 
+=== Code style suggestions
+[id="code_style_suggestions"]
+
+* In Perl files sort the use statements in this order from top to bottom:
+** "strict", "warnings" or other modules that provide static checks
+** All external modules and from "lib" folder
+** `use FindBin; use lib "$FindBin::Bin/lib";`
```
Comment (Martchus) [inline docs/Contributing.asciidoc:85]: It isn't always exactly this line.

File: docs/Contributing.asciidoc [hunk 2]
```diff
@@ -76,6 +76,17 @@ implications and how we can help each other to improve
...
+** internal test modules which provide early checks before other modules
+** other internal test modules
```
Comment (Martchus) [inline docs/Contributing.asciidoc:87]: These two only apply to tests.

Comment (okurz) [reply docs/Contributing.asciidoc:87]: Yes, that's true. But the complete list is only describing which modules should be put in which order. If a line does not apply just skip it. E.g. if there are no other "external modules" used and nothing from "lib" folder that line would also not apply. Do you think we have to change anything here?

File: docs/Contributing.asciidoc (hunk 1 above)
Comment (okurz) [reply docs/Contributing.asciidoc:85]: ok, I added "or similar to resolve internal modules".

File: docs/Contributing.asciidoc (hunk 2 above)
Comment (kalikiana) [reply docs/Contributing.asciidoc:88]: I'd say only the second to last *refers* to tests, so you could add "For tests:" there. No strong opinion, though.

**Review by kalikiana — APPROVED** [review]

**Review by Martchus — APPROVED** [review]

**Pull request merged**